lib/src/impl_stub/impl_stub.dart

```diff
@@ -0,0 +1,11 @@
+library impl_stub;
+
+import 'dart:typed_data';
+import 'dart:async';
+
+import '../impl_stub.dart'
+    if (dart.library.ffi) '../impl_ffi/impl_ffi.dart'
+    if (dart.library.js) '../impl_js/impl_js.dart' as impl;
+    
+
+part 'impl_stub.aescbc.dart';
```

Comment:
please end files with a line break.

See https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Reply:
Updated! thanks 🙌